src/screens/DataClearFilters.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import {
+  Data,
+  // DataClearFilters
```

Comment:
Can we uncomment this and update yarn.lock to get the latest stable version of grommet?